src/main/java/org/pih/petl/job/config/JobConfiguration.java

```diff
@@ -0,0 +1,186 @@
+package org.pih.petl.job.config;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.ValueNode;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.pih.petl.PetlUtil;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
```

Comment:
This is just a formal Object to represent a job's configuration, which was previously just a JsonNode on the object, and I moved all of the various methods that previously existed on PetlJobConfig for getting configuration values from this onto this class.  See below.